architecture/arm_security_extensions.rst

```diff
@@ -0,0 +1,65 @@
+.. _arm_security_extensions:
+
+#######################
+Arm Security Extensions
+#######################
+
+.. _bti:
+
+Branch Target Identification
+****************************
+
+Branch Target Identification (BTI) is an ARMv8.5 extension that provides
+Control Flow Integrity (CFI) around indirect branches and their targets, thus helping
+to limit the JOP (Jump Oriented Programming) attacks.
+
+With this extension, ARM8.5-A introduces Branch Target Instructions (BTIs).
+BTIs are also called launching pads. The processor can be configured so that
+indirect branches (BR and BLR) only allows target landing pad instructions.
+If the target of an indirect branch is not a landing pad, a Branch Target Exception 
+is generated.
+
+How to enable BTI for OP-TEE core
+==================================
+
+To make use of BTI in TEE core on CPU's that support it, enable the option 
+``CFG_CORE_BTI.``
+
+OP-TEE core makes use of some built-ins in the GCC/clang toolchains. So, in order
+to use the option ``CFG_CORE_BTI``, make sure that GCC toolchain has been built with 
+``--enable-standard-branch-protection`` is used else OP-TEE will fail to build.
+By default libraries such as libgcc.a are built with default flags (``-mbranch-protection=none``),
```

Comment:
_By default ... with default flags._
I guess one of the defaults can be removed.